include/mbedtls/psa_util.h

```diff
@@ -3,7 +3,7 @@
  *
  * \brief Utility functions for the use of the PSA Crypto library.
  *
- * \warning This function is not part of the public API and may
+ * \warning These functions are not part of the public API and may
  *          change at any time.
```

Comment:
As of 13a8c9ccaa983da92b4a0b8009828403c99dae52, only the RNG interface remains in this file. I think it's time to make it public. I don't anticipate it changing until 4.0 (when we might not need it anymore since code that needs a RNG can just can `psa_generate_random`).

Reply:
I'm mentioning `mbedtls_psa_get_random` [in the transition guide](https://github.com/Mbed-TLS/mbedtls/pull/7766/commits/c7b53f3ab7cd66a0153b0d9273f13f46261408ef).

Reply:
> should we announce the newly public functions?

We actually already announced them in the changelog back when we introduced them in 2.26.0. The functions themselves were documented as public, even though the file wasn't. I think we didn't pay attention to the warning at the top of the file when we added/reviewed the addition of `mbedtls_psa_get_random`.

> (I don't think the removal of internal functions needs announcing, we already removed some in previous PRs without an entry.)

Agreed.

Reply:
Ah, good to know, I wasn't aware we were already considering `mbedtls_psa_get_random` public.